OWNERS

```diff
@@ -1,24 +1,19 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - ahrtr            # Benjamin Wang <benjamin.ahrtr@gmail.com> <benjamin.wang@broadcom.com>
-  - fuweid           # Wei Fu <fuweid89@gmail.com>
-  - jmhbnz           # James Blair <jablair@redhat.com> <mail@jamesblair.net>
-  - serathius        # Marek Siarkowicz <siarkowicz@google.com> <marek.siarkowicz@gmail.com>
-  - spzala           # Sahdev Zala <spzala@us.ibm.com>
-reviewers:
-  - ivanvc           # Ivan Valdes <ivan@vald.es>
-  - siyuanfoundation # Siyuan Zhang <sizhang@google.com> <physicsbug@gmail.com>
+  - sig-etcd-chairs     # Defined in OWNERS_ALIASES
+  - sig-etcd-tech-leads # Defined in OWNERS_ALIASES
+  - fuweid              # Wei Fu <fuweid89@gmail.com>
 emeritus_approvers:
-  - bdarnell         # Ben Darnell <ben@bendarnell.com>
-  - fanminshi        # Fanmin Shi <fanmin.shi@gmail.com>
-  - gyuho            # Gyuho Lee <gyuhox@gmail.com>
-  - hexfusion        # Sam Batschelet <sbatsche@redhat.com>
-  - heyitsanthony    # Anthony Romano <romanoanthony061@gmail.com>
-  - jingyih          # Jingyi Hu <jingyih@google.com>
-  - jpbetz           # Joe Betz <jpbetz@google.com>
-  - mitake           # Hitoshi Mitake <h.mitake@gmail.com>
-  - philips          # Brandon Philips <brandon@ifup.org>
-  - ptabor           # Piotr Tabor <piotr.tabor@gmail.com>
-  - wenjiaswe        # Wenjia Zhang <wenjiazhang@google.com> <wenjia.swe@gmail.com>
-  - xiang90          # Xiang Li <xiangli.cs@gmail.com>
+  - bdarnell            # Ben Darnell <ben@bendarnell.com>
+  - fanminshi           # Fanmin Shi <fanmin.shi@gmail.com>
+  - gyuho               # Gyuho Lee <gyuhox@gmail.com>
+  - hexfusion           # Sam Batschelet <sbatsche@redhat.com>
+  - heyitsanthony       # Anthony Romano <romanoanthony061@gmail.com>
+  - jingyih             # Jingyi Hu <jingyih@google.com>
+  - jpbetz              # Joe Betz <jpbetz@google.com>
+  - mitake              # Hitoshi Mitake <h.mitake@gmail.com>
+  - philips             # Brandon Philips <brandon@ifup.org>
+  - ptabor              # Piotr Tabor <piotr.tabor@gmail.com>
+  - wenjiaswe           # Wenjia Zhang <wenjiazhang@google.com> <wenjia.swe@gmail.com>
+  - xiang90             # Xiang Li <xiangli.cs@gmail.com>
```

Comment:
Why is your editor adding space padding here?  Please remove this.

Reply:
Keeps the comments aligned, it looks better this way. You'll see what I mean when viewing the file without diff.